### PR TITLE
[2/2] Shield plugin

### PR DESCRIPTION
**Pull request by adamstegman**
https://www.pivotaltracker.com/story/show/111253006

Install the shield plugin and configure it with given roles and users.

The [logsearch-boshrelease](logsearch/logsearch-boshrelease) installs the package this job provides, but configuration must be done after restarting elasticsearch. So this job first waits for elasticsearch to start, then restarts it, then waits for shield to be set up, then configures shield.

### CODE REVIEW

File: jobs/elasticsearch-plugins-shield/templates/ctl [hunk 1]
```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
```
Comment (jeremiemv) [inline jobs/elasticsearch-plugins-shield/templates/ctl]: This is not something you wrote from scratch... right?

**Comment by jeremiemv** [comment]
Giving a :shipit: but would love to seat down for a review in person to have a better/deeper understanding :)

File: jobs/elasticsearch-plugins-shield/templates/ctl (hunk 1 above)
Comment (adamstegman) [inline jobs/elasticsearch-plugins-shield/templates/ctl]: This is what I was working on the whole time :yum:

Comment (jeremiemv) [inline jobs/elasticsearch-plugins-shield/templates/ctl]: Impressive!

**Comment by tomheinan** [comment]
:ship: :it:

**Pull request merged**